docs/index.html

```diff
@@ -322,6 +329,35 @@ <h5>YubiKey</h5>
         all have to be programmed with the same secret (see question above).</dd>
     </dl>
 
+    <h5>Browser extensions</h5>
+    <dl>
+        <dt id="faq-browser"><a href="#faq-browser">Does KeePassXC support browser extensions?</a></dt>
+        <dd>
+            Yes. KeePassXC supports KeePassHTTP-Connector as a legacy browser integration and a newer extension KeePassXC-Browser. While
+            KeePassHTTP-Connector still uses KeePassHTTP (which is going to be deprecated in the future), KeePassXC-Browser handles the
+            connection with KeePassXC through <a href="https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Native_messaging">native messaging</a>.
+            Both extensions are WebExtension compatible and also work with every Chromium-based browser.
+            You can download KeePassXC-Browser for <a href="https://addons.mozilla.org/en-US/firefox/addon/keepassxc-browser/">Mozilla Firefox</a> and
+            <a href="https://chrome.google.com/webstore/detail/keepassxc-browser/iopaggbpplllidnfmcghoonnokmjoicf">Google Chrome / Chromium / Vivaldi</a>.
```

Comment:
I think this is all a bit to technical. Users won't understand that much tech babble.